s4e-web/src/app/views/map-view/state/product/product.factory.spec.ts

```diff
@@ -1,5 +1,11 @@
 import * as Factory from 'factory.ts';
-import {Product} from './product.model';
+import { Product, ProductCategory } from './product.model';
+
+const ProductCategoryFactory = Factory.makeFactory<ProductCategory>({
+  id: Factory.each(i => i),
+  label: Factory.each(i => `Product category #${i}`),
+  url: Factory.each(i => `http://test.com/#${i}`),
```

Comment:
Wouldn't the actual values returned by the API be like: `categories/icon_${i}.svg`?

Reply:
That doesn't matter